server/src/main/java/com/cloud/capacity/CapacityManagerImpl.java

```diff
@@ -665,11 +665,10 @@ public void updateCapacityForHost(final Host host, final Map<Long, ServiceOfferi
             Map<String, String> vmDetails = _userVmDetailsDao.listDetailsKeyPairs(vm.getId());
             String vmDetailCpu = vmDetails.get("cpuOvercommitRatio");
             String vmDetailRam = vmDetails.get("memoryOvercommitRatio");
-            if (vmDetailCpu != null) {
-                //if vmDetail_cpu is not null it means it is running in a overcommited cluster.
-                cpuOvercommitRatio = Float.parseFloat(vmDetailCpu);
-                ramOvercommitRatio = Float.parseFloat(vmDetailRam);
-            }
+            //if vmDetail_cpu is not null it means it is running in a overcommitted cluster.
```

Comment:
My only suggestion would be to merge these two comments into one, at line 668.
Example:
```
//if vmDetailCpu or vmDetailRam is not null it means it is running in an overcommitted cluster.
```

Reply:
@GabrielBrascher @DaanHoogland done